test/com/facebook/buck/android/testdata/android_project/kotlin/com/sample/lib/JavaClass.java

```diff
@@ -0,0 +1,4 @@
+package com.sample.lib;
+
+class JavaClass {
```

Comment:
can we update this to be similar to the library test where we have bi-directional calls?